docs/docs/quickstart/installation/installation.md

```diff
@@ -11,21 +11,24 @@ include path.
 
 1. Copy `graaflip` to your project.
```

Comment:
[very minor]: This typo was here before, but maybe we could fix it now that we are touching this file
```suggestion

1. Copy `graaflib` to your project.
```